WordPress/Classes/ViewRelated/Post/PostSettingsViewController.m

```diff
@@ -431,13 +473,34 @@ - (NSString *)tableView:(UITableView *)tableView titleForHeaderInSection:(NSInte
     } else if (sec == PostSettingsSectionShare && [self numberOfRowsForShareSection] > 0) {
         return NSLocalizedString(@"Jetpack Social", @"Label for the Sharing section in post Settings. Should be the same as WP core.");
 
+    } else if (sec == PostSettingsSectionDisabledTwitter) {
+        return NSLocalizedStringWithDefaultValue(@"postSettings.section.disabledTwitter.header",
+                                                 nil,
+                                                 [NSBundle mainBundle],
+                                                 @"Twitter Auto-Sharing Is No Longer Available",
+                                                 @"Section title for the disabled Twitter service in the Post Settings screen");
```

Comment:
Just curious, is `NSLocalizedStringWithDefaultValue` being used here so we can have a key?

Reply:
Yes 👍🏼 . The direct equivalent of Swift's `NSLocalizedString(key, value: value, comment: comment)` is
```objc
NSLocalizedStringWithDefaultValue(key, nil, NSBundle.mainBundle, value, comment)
```
in Objective-C.